.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+  - 2.7
```

Comment:
Should add all versions that mongo-connector supports. So add 2.6, 3.3, 3.4. We might as well also add 3.5 since travis already supports it:

http://docs.travis-ci.com/user/languages/python/